MAUI/DataGrid/grid-events.md

```diff
@@ -0,0 +1,201 @@
+* [CellLongPress](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellLongPress) : Called when a long-press gesture with a primary button has been recognized for a cell.
+* [CellRightTapped](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_CellRightTapped) : Called when a right-click mouse gesture has been recognized on a cell.
+
+### CellTapped event
+This event will be triggered while tapping a cell in the DataGrid. This event has the [DataGridCellTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCellTappedEventArgs.html) as arguments.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   CellTapped="dataGrid_CellTapped"
+                   ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void dataGrid_CellTapped(object sender,DataGridCellTappedEventArgs e)
+{
+    var rowIndex = e.RowColumnIndex.RowIndex;
+    var rowData = e.RowData;
+    var columnIndex = e.RowColumnIndex.ColumnIndex;
+    var column = e.Column;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### CellDoubleTapped event
+
+This event will be triggered while double tapping a cell in the DataGrid. This event has the [DataGridCellDoubleTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCellDoubleTappedEventArgs.html) as arguments. 
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   CellDoubleTapped="dataGrid_CellDoubleTapped"
+                   ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void dataGrid_CellDoubleTapped(object sender, DataGridCellDoubleTappedEventArgs e)
+{
+    var rowIndex = e.RowColumnIndex.RowIndex;
+    var rowData = e.RowData;
+    var columnIndex = e.RowColumnIndex.ColumnIndex;
+    var column = e.Column;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### CellLongPress event
+
+This event will be triggered while long pressing a cell in the DataGrid. This event has the [DataGridCellLongPressEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCellLongPressEventArgs.html) as arguments. 
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   CellLongPress="dataGrid_CellLongPress"
+                   ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void dataGrid_CellLongPress(object sender,DataGridCellLongPressEventArgs e)
```

Comment:
Space should be given between two arguments in a method,

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/7f4a85a6-ec24-4e09-857e-7b054e0683b3)
Had updated as discussed.